nil/internal/consensus/ibft/ibft.go

```diff
@@ -74,6 +84,11 @@ func (i *backendIBFT) InsertProposal(proposal *protoIBFT.Proposal, committedSeal
 	if err != nil {
 		return
 	}
+	i.logger.Debug().
+		Uint64(logging.FieldBlockNumber, proposalBlock.PrevBlockId.Uint64()+1).
+		Uint64(logging.FieldRound, proposal.Round).
+		Uint32(logging.FieldShardId, uint32(i.shardId)).
```

Comment:
This field is already defined here

https://github.com/NilFoundation/nil/blob/b3a123a44ede3cffe12b327f0d1ef762db9d6ca5/nil/internal/consensus/ibft/ibft.go#L99